tapdb/sqlc/queries/universe.sql

```diff
@@ -159,14 +159,49 @@ INSERT INTO universe_events (
 );
 
 -- name: QueryUniverseStats :one
-WITH num_assets As (
-    SELECT COUNT(*) AS num_assets
+WITH stats AS (
+    SELECT total_asset_syncs, total_asset_proofs
+    FROM universe_stats
+), num_groups AS (
+    SELECT id
     FROM universe_roots
+    WHERE group_key IS NOT NULL
+), num_assets AS (
+    SELECT sum, hash_key
+    FROM mssmt_nodes nodes
+    JOIN mssmt_roots roots
+      ON nodes.hash_key = roots.root_hash AND
+             nodes.namespace = roots.namespace
+    JOIN universe_roots uroots
+      ON roots.namespace = uroots.namespace_root
+), aggregated AS (
+    SELECT COALESCE(SUM(stats.total_asset_syncs), 0) AS total_syncs,
+           COALESCE(SUM(stats.total_asset_proofs), 0) AS total_proofs,
+           0 AS total_num_groups,
+           0 AS total_num_units,
+           0 AS total_num_tranches
+    FROM stats
+    UNION ALL
+    SELECT 0 AS total_syncs,
+           0 AS total_proofs,
+           COALESCE(COUNT(num_groups.id), 0) AS total_num_groups,
+           0 AS total_num_units,
+           0 AS total_num_tranches
+    FROM num_groups
+    UNION ALL
+    SELECT 0 AS total_syncs,
+           0 AS total_proofs,
+           0 AS total_num_groups,
+           COALESCE(SUM(num_assets.sum), 0) AS total_num_units,
+           COALESCE(COUNT(num_assets.hash_key), 0) AS total_num_tranches
```

Comment:
I think rename this to `num_total_assets` perhaps? Will also have a slightly different meaning for transfers as well. Since that'll be the total number of transfers that've happend. For the events, we have the type that can be used.